corehq/motech/repeaters/tests/test_tasks.py

```diff
@@ -558,3 +559,65 @@ def test_update_repeater_does_nothing_on_none(self, mock_get_repeater, __):
 
         mock_repeater.set_backoff.assert_not_called()
         mock_repeater.reset_backoff.assert_not_called()
+
+
+class TestGetWaitDurationSeconds(TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.repeater = FormRepeater.objects.create(
+            domain=DOMAIN,
+            connection_settings=ConnectionSettings.objects.create(
+                domain=DOMAIN,
+                url='http://www.example.com/api/'
+            ),
+        )
+
+    def test_repeat_record_no_attempts(self):
+        five_minutes_ago = datetime.utcnow() - timedelta(minutes=5)
+        repeat_record = RepeatRecord.objects.create(
+            repeater=self.repeater,
+            domain=DOMAIN,
+            payload_id='abc123',
+            registered_at=five_minutes_ago,
+        )
+        wait_duration = _get_wait_duration_seconds(repeat_record)
+        self.assertEqual(wait_duration, 300)
```

Comment:
Could there be a race condition (resulting in a flaky test) here if something took an extra second or two (like garbage collection) between when `repeat_record` is created and when `_get_wait_duration_seconds` is run? Seems unlikely, but not impossible. Maybe consider using something like `freezegun.freeze_time`?

The same applies to other tests in this class as well.